### ISSUE TITLE
Wiki page names prevent repo cloning

### ISSUE THREAD

**Issue by pcolmer** [Windows-XAML/Template10#1213]
I'd like to clone the wiki repository so that I have an offline copy of the documentation. However, the use of | in the page names prevents those pages from being created locally on a Windows filing system.

Would it be OK if I renamed the pages so that a hyphen was used instead? I'll undertake to ensure that all hyperlinks continue to work ...

**Comment by JerryNixon** [comment]
This would be fine

**Comment by pcolmer** [comment]
Just a couple of comments:
- It turns out I couldn't use a hyphen in the page title instead of |. GitHub uses the hyphen in filenames to represent spaces, so that doesn't work. I've removed the "Docs | " and "Welcome | " bits completely.
- The Controls page makes reference to a PageHeader page which doesn't seem to exist. What I found particularly curious is that there is an anchor reference to pageheader-and-the-hamburgermenu, suggesting that the page used to exist or the author had a clear idea of what that page was going to say.
